_emails/us/florida/tallahassee

```diff
@@ -0,0 +1,40 @@
+---
+title: Tallahassee, FL
+permalink: "/tallahassee"
+name: Letter to Mayor and City Council
+city: Tallahassee
+state: FL
+layout: email
+recipients:
+- mayor@talgov.com
+- dianne.williams-cox@talgov.com
+- elaine.bryant@talgov.com
+- curtis.richardson@talgov.com
+- jeremy.matlow@talgov.com
+subject: '[*** INSERT UNIQUE SUBJECT LINE***]'
+body: |-
+  Dear Mayor Dailey and Tallahassee City Commissioners,
+
+  My name is [NAME] and I am a resident of [NEIGHBORHOOD]. I request that the Mayor and City Commission support our community and hear our call for:
+
+  - Efforts to defund and de-escalate the police dept. and reallocate the money to programs and city-led initiatives that support education, rehabilitation, public health, and community-oriented activities.
+
+  - Rejection of expanded and militarized police budgets.
+
+  - Efforts to improve oversight and independent investigations to hold individual law enforcement officers and police departments accountable.
+
+  - Efforts to establish an all-civilian review board with the authority to investigate incidents of police misconduct, to ensure community level oversight accountability and discipline the reaction of police officers.
+
+  - The adoption of sound and unbiased law enforcement policies at all levels of government that reduce the disparate impact of police brutality, racial profiling, and use of force on BIPOC and other historically marginalized communities.
+
+  - Redirection of money away from the Tallahassee PD and into the following social services: Community Development, Public Transportation, Education, Parks & Recreation, targeted support for marginalized communities.
+
+  - Action to address the racial wealth gap in Tallahassee and decrease the spread of rampant gentrification and rising housing costs that disproportionately target BIPOC residents.
+
+  I have noticed that in the FY20 approved budget the police department's budget is over $59 million dollars. That is over twice the amount for parks and recreation, and, critically, nearly three times the amount for StarMetro Operating Fund. In light of recent events, I believe it is critical that policing be deprioritized relative to spending on social programs that are actually proven to promote safe and equitable communities.
+
+  I request that an emergency meeting be called before the fiscal year goes into effect in order to reallocate these funds to resources the citizens can benefit from.
+
+  Thank you for your time,
+  [YOUR NAME] [YOUR ADDRESS] [YOUR EMAIL] [YOUR PHONE NUMBER]
```

Comment:
```suggestion
  - Efforts to defund and de-escalate the police department, and to reallocate money to programs and city-led initiatives that support education, rehabilitation, public health, and community-oriented activities.

  - Rejection of expanded and militarized police budgets.

  - Efforts to improve oversight and independent investigations to hold individual law enforcement officers and police departments accountable for misconduct.

  - Redirection of money away from the Tallahassee PD, and into Community Development, Public Transportation, Education, Parks & Recreation, as well as targeted support for marginalized communities.

  I have noticed that in the FY20 approved budget the police department's budget is over $59 million. That is over twice the amount for parks and recreation, and, critically, nearly three times the amount for StarMetro Operating Fund. In light of recent events, I believe it is critical that policing be deprioritized relative to spending on social programs that are actually proven to promote safe and equitable communities.

  I request that an emergency meeting be called before the fiscal year goes into effect in order to reallocate these funds to resources that community members can benefit from.

  Thank you for your time,

  [YOUR NAME]

  [YOUR ADDRESS]

  [YOUR EMAIL]

  [YOUR PHONE NUMBER]
```

Reply:
Man, GitHub is not that great at displaying diffs here, as far as I can tell the signoff was just split into separate lines? Looks good to me...